Cargo.toml

```diff
@@ -19,9 +19,9 @@ crate-type = ["cdylib"]
 petgraph = "0.5.1"
 fixedbitset = "0.2.0"
 hashbrown = "0.7"
-numpy = "0.9.0"
+numpy = "0.10.0"
```

Comment:
It's not up yet, they've merged the version bump: https://github.com/PyO3/rust-numpy/commit/60bdaad175b5dc42588f54a7ac450807be517bd8 but haven't pushed the release yet. This is blocked until its released because without it the pyo3 version it uses doesn't match what retworkx uses.